packages/http-server-js/src/scripts/scaffold/data-mocks.ts

```diff
@@ -19,13 +23,13 @@ import { parseCase } from "../../util/case.js";
  * @returns A JavaScript string representation of the mock data
  * @throws Error if a property cannot be mocked
  */
-function mockModel(type: Model): string {
+function mockModel(ctx: JsContext, module: Module, type: Model): string {
```

Comment:
Is `ctx` passed for future changes? I looked but didn't see it actually getting used anywhere with these changes (I could have missed it)

Reply:
Ah, you're right it's never used. I only needed `module`, but I'm simply programmed to always pass `ctx` and `module` in a pair.

Reply:
I think I'll leave this as passing both ctx and module. I think I will need both to add support for decimal128/decimal in an upcoming PR.